rts/lra-examples/lra-jwt/wildfly/lra-coordinator/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns="http://java.sun.com/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd" version="3.0">
+
+    <security-constraint>
+        <web-resource-collection>
+            <web-resource-name>Protect all application</web-resource-name>
+            <url-pattern>/lra-coordinator/*</url-pattern>
+        </web-resource-collection>
+        <auth-constraint>
+            <role-name>admin</role-name>
+        </auth-constraint>
+    </security-constraint>
+    <security-constraint>
+        <web-resource-collection>
+            <web-resource-name>participant</web-resource-name>
+            <url-pattern>/lra-coordinator/start</url-pattern>
```

Comment:
Yes, this quickstart do not follow the recommendation because, this is to just give an idea to users how to use the JWT mechanism for LRA coordinator deployed in Wildfly server.

It's user's responsibility to follow the recommendation.

Reply:
Ok, I'm not fully satisfied with this approach but I'm fine with your decision. Please, specify the details - what is secured, how it's configured, where it'd defined, how to follow with the recommendation to enhance the quickstart in the README.

Reply:
okay